packages/manager/modules/pci/src/projects/project/storages/databases/add/add.html

```diff
@@ -259,6 +259,7 @@
                 data-lab="$ctrl.lab"
                 data-ng-if="!$ctrl.lab.isActivated()"
                 data-on-accept="$ctrl.acceptLab(accepted)"
+                data-on-contract-click="$ctrl.clickOnContract(contract)"
```

Comment:
This method does not seem to be taking any argument...

```suggestion
                data-on-contract-click="$ctrl.clickOnContract()"
```

Reply:
Removed: https://github.com/ovh/manager/commit/4ced9b709afd77be339062029326f63e1808eeca